lib/command/build_image.rb

```diff
@@ -36,9 +36,14 @@ def call # rubocop:disable Metrics/MethodLength
 
       cp.image_build(image_url, dockerfile: dockerfile,
                                 docker_args: config.args,
-                                build_args: build_args)
+                                build_args: build_args,
+                                push: false)
```

Comment:
We can remove `push` from `image_build`, since it was only being used here. I can't think of another scenario where it would be used.